pkg/services/ngalert/api/tooling/definitions/prom.go

```diff
@@ -152,9 +152,10 @@ type AlertingRule struct {
 	// required: true
 	Name string `json:"name,omitempty"`
 	// required: true
-	Query         string  `json:"query,omitempty"`
-	Duration      float64 `json:"duration,omitempty"`
-	KeepFiringFor float64 `json:"keepFiringFor,omitempty"`
+	Query                 string   `json:"query,omitempty"`
+	QueriedDatasourceUIDs []string `json:"queriedDatasourceUIDs,omitempty"`
```

Comment:
not entirely sure if this has been bike shed yet; but it took me a while to figure out that this is where we store the list of data query IDs that appear in the alert rule definition (initially I thought this was something that was computed at run-time when the alert rule had been evaluated).

can we try and find a clearer name?

Reply:
maybe `DataSourceUIDsFromRuleDefinition`? 😬

Reply:
Wouldn't it be confusing in the other direction? Most of the fields in this struct come from the rule definition, but only one would be called this way